src/main/java/org/gridsuite/shortcircuit/server/service/ShortCircuitService.java

```diff
@@ -40,17 +47,39 @@
  */
 @Service
 public class ShortCircuitService extends AbstractComputationService<ShortCircuitRunContext, ShortCircuitAnalysisResultService, ShortCircuitAnalysisStatus> {
-
     private static final Logger LOGGER = LoggerFactory.getLogger(ShortCircuitService.class);
     public static final String GET_SHORT_CIRCUIT_RESULTS_MSG = "Get ShortCircuit Results {} in {}ms";
 
-    public ShortCircuitService(NotificationService notificationService, UuidGeneratorService uuidGeneratorService, ShortCircuitAnalysisResultService resultService, ObjectMapper objectMapper) {
+    public static final List<VoltageRange> CEI909_VOLTAGE_PROFILE = List.of(
+        new VoltageRange(10.0, 199.99, 1.1),
+        new VoltageRange(200.0, 299.99, 1.09),
+        new VoltageRange(300.0, 500.0, 1.05)
+    );
+
+    private final ParametersRepository parametersRepository;
+
+    public ShortCircuitService(final NotificationService notificationService, final UuidGeneratorService uuidGeneratorService,
+                               final ShortCircuitAnalysisResultService resultService, final ParametersRepository parametersRepository,
+                               final ObjectMapper objectMapper) {
         super(notificationService, resultService, objectMapper, uuidGeneratorService, null);
+        this.parametersRepository = parametersRepository;
     }
 
+    public UUID runAndSaveResult(UUID networkUuid, String variantId, String receiver, UUID reportUuid, String reporterId, String reportType,
+                                 String userId, String busId, final Optional<UUID> parametersUuid, final Optional<ShortCircuitParametersInfos> givenParameters) {
+        ShortCircuitParameters parameters = givenParameters.or(() -> parametersUuid.flatMap(parametersRepository::findById)
+                                                                                   .map(ShortCircuitService::fromEntity))
+                                                           .map(ShortCircuitParametersInfos::parameters)
+                                                           .orElseGet(() -> getDefaultDtoParameters().parameters());
+        parameters.setWithFortescueResult(StringUtils.isBlank(busId));
```

Comment:
This parameter is stored in our entity, should we really override it ? If so it's not necessary to store it

Reply:
That is how it's done in study-server, so to see with the PO why.

Reply:
Only the "One bus" analysis is run with fortescueResult set to true

Reply:
> This parameter is stored in our entity, should we really override it ? If so it's not necessary to store it

Looked with @etiennehomer and has been removed from entity.